fedgov-cv-ISDcreator-webapp/src/main/webapp/js/config.js

```diff
@@ -428,7 +428,7 @@ var help_notes = [
 		max: "NA",
 		min: "NA",
 		units: "N/A",
-		description: "This data frame defines the Object Identifier (OID) of the Road Authority. Road authority ID is mandatory if region id (roadRegulatoryID) is set to 0."
+		description: "Road Authority ID, contained in the Base Layer of the RGA has the same value as that of the organization responsible for providing the DS content. If IEEE 1609.2 security is used (see 6.8), this will be contained in the ‘Operating Organization ID’ field of the security certificate used to sign the RGA. How this is obtained if other security mechanisms are used is outside the scope of this report."

```

Comment:
Not sure which is 6.8 section. Where this descript come from?

Reply:
I wasn't sure, this description that I put was in my PR so I just copied/pasted into here. I can change it to something else but I wasn't sure who to ask about this